packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -102,102 +102,241 @@ export const Default = {
+};
+const stateLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 4",
+  borderStyle: "solid none none none",
+  borderWidth: "1px",
+  justifySelf: "stretch",
+  margin: "20px 0 10px 0",
+  paddingTop: "10px",
+};
+const hoverLabel = {
+  fontWeight: "normal",
+  gridColumn: "1 / 2",
+  alignSelf: "end",
+  margin: "0 0 20px 0",
+};
+const typeLabel = {
+  fontWeight: "normal",
+  gridColumn: "1 / 3",
+  margin: "10px 0",
+};
+const iconLabel = {
+  fontWeight: "normal",
+  margin: "0 0 10px 0",
+};
+
+const SDS_STYLES = ["rounded", "square"];
+const COLORS = [
+  "primary",
+  "info",
+  "success",
+  "warning",
+  "error",
+  "gray",
+  "beta",
+  ["#A52A2A", "#ffa07a", "#ffe74a"],
+];
+const PSEUDO_STATES = ["default", "hover", "active", "focus-visible"];
+const HOVERABLE_OPTIONS = [true, false];
+const SDS_TYPES = ["primary", "secondary"];
+const ICON_OPTIONS = [
+  undefined,
+  <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
+];
 
 export const LivePreview = {
   args: {
+    sdsStyle: SDS_STYLES[0],
+    color: COLORS[0],
+    hover: HOVERABLE_OPTIONS[0],
+    sdsType: SDS_TYPES[0],
     label: "Label",
   },
   parameters: {
     snapshot: {
       skip: true,
     },
   },
-  render: (args: Args) => <LivePreviewDemo {...args} />,
+
+  render: (props: Args): JSX.Element => {
+    // loop through all SDS_STYLES
+    return (
+      <>
+        {SDS_STYLES.map((sdsStyle) => {
+          return <TagStyle sdsStyle={sdsStyle} key={sdsStyle} />;
+        })}
+      </>
+    );
+
+    // loop through all COLORS + create headers for SDS_STYLES
+    function TagStyle({ sdsStyle }) {
+      return (
+        <div style={styleLevel}>
+          <h1 style={styleLabel}>
+            Style: <b>{sdsStyle}</b>
+          </h1>
+          {COLORS.map((color) => {
+            return (
+              <TagHoverableOption
+                sdsStyle={sdsStyle}
+                color={color}
+                key={color}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all PSEUDO_STATES + create headers for COLORS
+    function TagHoverableOption({ sdsStyle, color }) {
+      return (
+        <div style={displayContents}>
+          <h2 style={colorLabel}>
+            Color: <b>{color[0].length === 1 ? color : "custom"}</b>
+          </h2>
+          {PSEUDO_STATES.map((state) => {
+            return (
+              <TagColor
+                sdsStyle={sdsStyle}
+                color={color}
+                state={state}
+                key={state}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all HOVERABLE_OPTIONS + create headers for PSEUDO_STATES
+    function TagColor({ sdsStyle, color, state }) {
+      return (
+        <div style={displayContents}>
+          <h3 style={stateLabel}>
+            Pseudo state: <b>{state}</b>
+          </h3>
+          {HOVERABLE_OPTIONS.map((hover, hoverIndex) => {
```

Comment:
In order to properly stack the the hoverable iterations of the Tag component ("Hoverable: yes" and "Hoverable: no") within the CSS grid for easier visual comparison, the index of the items in this `.map()` loop needs to be passed all the way down to the last loop. If this loop's index is zero, the sdsType and the Icon headings are rendered, but not otherwise (see lines 308 and 317 below). This allows multiple rows of tags (just two in this case) to be stacked with just one sdsType and Icon headings as needed, applying to the columns created by the Tag components' placements within the CSS grid.